### ISSUE TITLE
Assertion failed: Capacity() - Size() > 0, file C:\buildbot\slave\builds\engine-win32-64-beta\build\tmp\dynamo_home\sdk\include\dmsdk/dlib/array.h, line 464

### ISSUE THREAD

**Issue by subsoap** [defold/editor2-issues#2464]
This is happening when running builds, seemingly randomly.

It closes the game and shows this in console. Rebuilding works usually though?

**Comment by subsoap** [comment]
```
Assertion failed: Capacity() - Size() > 0, file C:\buildbot\slave\builds\engine-win32-64-beta\build\tmp\dynamo_home\sdk\include\dmsdk/dlib/array.h, line 464
```

Happened again. Maybe something to do with loading collections?

**Comment by mathiaswking** [comment]
Any luck finding a `_crash` file?

**Comment by subsoap** [comment]
No _crash file created recently enough to be relevant. Searched entire drive to be sure, Oldest one I have is from a week ago. Might still be useful for something else so here it is...

[_crash.zip](https://github.com/defold/editor2-issues/files/2850672/_crash.zip)

Was in C:\Users\Brian\AppData\Roaming\Defold for me

**Comment by danengelbrecht-king** [comment]
Created jira DEF-3774

**Comment by AGulev** [comment]
Issue was fixed in 1.2.147 https://forum.defold.com/t/defold-1-2-147-has-been-released/43851
